### PR TITLE
feat: make payload size configurable

### PR DESCRIPTION
**Pull request by syncush**
| Question                | Answer                                                                          |
| ---------------- | -------------------------------------------------------------------------- |
| Bug fix         | ✖                                                                        |
| New feature     | ✔                                                                        |
| Breaking change | ✖                                                                        |
| Deprecations    | ✖                                                                        |
| Documentation   | ✖                                                                        |
| Tests added     | ✖                                                                        |
| Chore            | ✖                                                                       |

### CODE REVIEW

File: src/serverBuilder.ts [hunk 1]
```diff
@@ -40,7 +40,7 @@ export class ServerBuilder {
   }
 
   private registerPreRoutesMiddleware(): void {
-    this.serverInstance.use(bodyParser.json());
+    this.serverInstance.use(bodyParser.json(this.config.get<bodyParser.Options>('server.request.payload')));
```
Comment (CptSchnitz) [inline src/serverBuilder.ts:43]: we sure we want the entire object as parameter and not just the relevant value?

File: config/default.json [hunk 2]
```diff
@@ -9,6 +9,11 @@
     "level": "info"
   },
   "server": {
-    "port": "8080"
+    "port": "8080",
+    "request": {
+      "payload": {
+        "limit": "10mb"
```
Comment (CptSchnitz) [inline config/default.json:15]: do we also want to expose this as an environment variable?

**Review by CptSchnitz — CHANGES_REQUESTED** [review]

File: config/default.json (hunk 2 above)
[the remark this answers is not in the document]
Comment (syncush) [reply config/default.json:15]: 1Mb ?

File: src/serverBuilder.ts (hunk 1 above)
Comment (syncush) [reply src/serverBuilder.ts:43]: all options parameters have default values, do you still think we should accept only the payload size?

File: config/default.json (hunk 2 above)
[the remark this answers is not in the document]
Comment (CptSchnitz) [reply config/default.json:15]: seems good.

Comment (syncush) [reply config/default.json:15]: done

[the remark this answers is not in the document]
Comment (syncush) [reply config/default.json]: done

**Review by CptSchnitz — APPROVED** [review]

**Pull request merged**